mbuild/tests/test_compound.py

```diff
@@ -1193,3 +1193,51 @@ def test_visualize_nglview(self, ethane):
         nglview = import_("nglview")
         vis_object = ethane._visualize_nglview()
         assert isinstance(vis_object.component_0, nglview.component.ComponentViewer)
+
+    def test_element(self):
+        from mdtraj.core.element import get_by_symbol
+        na = get_by_symbol("Na")
+        na_compound = mb.Compound(element="Na")
+        assert na_compound.element == na
+        na_compound = mb.Compound(element="NA")
+        assert na_compound.element == na
+        na_compound = mb.Compound(element="na")
+        assert na_compound.element == na
+        co_compound = mb.Compound(element="Co")
+        assert co_compound.element != na
+
+        na_compound_clone = mb.clone(na_compound)
+        assert na_compound_clone.element == na
+        container = mb.Compound()
+        container.add(na_compound)
+        container.add(na_compound_clone)
+        for child in container.children:
+            assert child.element == na
+
```

Comment:
Can we add a few lines to test the setter and getter for Compound's element (mainly setter)? (So setting the element after setting a blank Compound and make sure it's what we expect)